### PR TITLE
PROC-1295: Fix uploading web fragments

### PR DESCRIPTION
**Pull request by zacharygoodwin**

### CODE REVIEW

File: proctor-tomcat-deps/build.gradle [hunk 1]
```diff
@@ -4,4 +4,20 @@ indeedLibrary.name = 'proctor-tomcat-deps'
 dependencies {
     api libs.tomcatElApi
     api libs.tomcatJasperEl
-}
\ No newline at end of file
+}
+
+task editPom(type: Exec) {
+    commandLine './editPom.sh'
```
Comment (vanhaxl) [inline proctor-tomcat-deps/build.gradle:10]: Just try to understand what is the file ./build/publications/maven/pom-default.xml
and why do we need to delete these 2 lines?

Comment (zacharygoodwin) [reply proctor-tomcat-deps/build.gradle:10]: Gradle build will create the pom-default.xml. Gradle publish will move this change the name and upload to nexus/maven. 

Prior to switching to gradle maven allows no version being defined in the pom.xml but gradle requires and outputs a version in the pom.xml Defining a version breaks repos which also defines the tomcat libraries in their build. This is a hack to fix it. Ideal future is getting rid of shading and investigate upgrading tomcat deps. I will DM you links to view to help make more sense.

**Review by vanhaxl — APPROVED** [review]
LGTM

File: proctor-store-git/build.gradle [hunk 2]
```diff
@@ -19,4 +23,44 @@ dependencies {
...
+
+shadowJar.dependsOn(':proctor-store:shadowJar')
+shadowJar.dependsOn(':proctor-common:shadowJar')
+
+// Need to remove all dependenceis that get added or will cause issues upstream when projects include
+// Plan to remove in PROC-1290: Remove Proctor Shading of Javax El libraries
+shadowJar {
+    archiveClassifier.set('')
+    dependencies {
+        exclude(dependency('org.springframework:spring-core:5.3.19'))
```
Comment (vanhaxl) [inline proctor-store-git/build.gradle:37]: Do we need to maintain the version of the dependency that it excludes?

Comment (zacharygoodwin) [reply proctor-store-git/build.gradle]: Removed in all excludes

**Pull request merged**